api/datastore/sql/migrations/10_add_app_id.go

```diff
@@ -22,7 +22,7 @@ var sqlStatements = [...]string{`CREATE TABLE IF NOT EXISTS routes (
 	type varchar(16) NOT NULL,
 	headers text NOT NULL,
 	config text NOT NULL,
-	annotations text NOT NULL,
+	annotations text,
```

Comment:
we need an additional migration (`ALTER TABLE`...) on top of this anyway to handle changing this field to NOT NULL for any database that has run this, don't we? seems better to encapsulate this change there instead of modifying old migration in addition to new one. not sure, just checking.

Reply:
bump @zootalures

Reply:
Bugger, thanks for spot - added that and rebased